_component_design/badge.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Badges
+parent: Notifications
+layout: component
+category: Components
+usage: >
+  Notification badges are used to let the user know that there is new activity that hasn’t yet been acknowledged. It is used to encourage them to click into that section to see the details of the activity. 
+preview-image: preview-images/badges.svg
+resource: true
+status: stable
+last-modified: 2017-11-01
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### When to use
+
+Notification badges can be used in navigation menus, in tabs, or in the body of a page. They are used to signal a new notification, alert, message, chat, comment, or other type of change to an area of the site. Depending on the use case, there may or may not be a number of notifications written in the badge itself. Studies of social apps report that notification badges can trigger dopamine releases due to social rewards, but overuse can make the user numb to them, so use wisely.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The notification badge pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/notifications/badges/badges-hero.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+### Best Practices
+- Clicking on the badge takes the user into the section to which the badge is applied and the badge is removed.
```

Comment:
"Clicking on the badge takes the user into the section to which the badge is applied. Once the badge is clicked, remove the badge."

(Also, what do you mean "takes the user into the section to which the badge is applied"? Wouldn't the user be in the section where the badge is to see it? Do you mean that the action of clicking on the notification that comes up from the badge makes the view jump to the section that triggered the notification? I find this wording very confusing.)

Reply:
Agreed, this is confusing. I'll reword with Lane (writer)